### DISCUSSION TITLE
Is there any way that I can import this for replit?

### DISCUSSION THREAD

**Discussion by AndyBandyBoi** [nextcord/nextcord#296] [Q&A] [answered]
I wanna import nextcord into replit but it says that nextcord isnt a thing is there any other way that I can import it?

**Comment by VincentRPS** [comment]
nextcode or nextcord?

**Reply by AndyBandyBoi** [reply]
nextcord sorry

**Reply by alex-101** [reply]
Use the shell
`pip3 install -U git+https://github.com/nextcord/nextcord.git#master`
Replit has a few modules already, but you can always add your own.

**Reply by ooliver1** [reply]
* replit uses poetry so pip would be ignored, `poetry add git+https://github.com/nextcord/nextcord`

**Reply by alex-101** [reply]
Mine seemed to work, I don't know though.
```python
import nextcord
print(nextcord)
## <module 'nextcord' from '/opt/virtualenvs/python3/lib/python3.8/site-packages/nextcord/__init__.py'>
```

**Comment by TAG-Epic** [comment, marked as answer]
Open the replit terminal and run `poetry add nextcord`
